doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -15,6 +15,7 @@
 * [Ordering between:$
  different attribute value types](#ordering-between-different-attribute-value-types)
 * [Initial notifications](#initial_notifications)
+* [Oneshot Subscription](#Oneshot subscriptions)
```

Comment:
#Oneshot subscriptions -> #oneshot_subscriptions

Reply:
fixed in a0695b2